iFuture/src/containers/SignupPage/index.js

```diff
@@ -0,0 +1,199 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { signup } from '../../actions/user'
+
+import Appbar from '../../components/Appbar';
+
+import * as S from './styles'
+import { InputAdornment } from '@material-ui/core';
+
+const logo = <img src={require("../../assets/logo-future-eats-invert.svg")} alt='Voltar' />
```

Comment:
Seria melhor que fosse um componente, para garantir que só está sendo carregado dentro do `render()`

Assim já resolve:
```jsx
 const Logo = () => <img src={require("../../assets/logo-future-eats-invert.svg")} alt='Voltar' />
```

e quando for usar dentro do `render`:

```jsx
  <Logo />
```